src/main/java/uk/gov/digital/ho/hocs/info/api/TeamResource.java

```diff
@@ -100,14 +101,14 @@ public ResponseEntity deleteTeamPermissions(@PathVariable String teamUUID, @Requ
 
     @GetMapping(value = "/team/{teamUUID}/move_options", produces = APPLICATION_JSON_UTF8_VALUE)
     public ResponseEntity<Set<TeamDto>> getMoveToAnotherTeamOptions(@PathVariable UUID teamUUID) {
-        Team team = teamService.getTeam(teamUUID); // todo: this
-        final Unit unit = team.getUnit();
+        Team team = teamService.getTeam(teamUUID);
+
+        Set<Team> teams = team.getUnit().isAllowBulkTeamTransfer() ?
+                teamService.findActiveTeamsByUnitUuid(team.getUnit().getUuid()) : Collections.emptySet();
```

Comment:
As I understand this change previously an active team would be returned providing there was a team unit and uuid, but now this will only be returned if that exists and also if the bulk transfer is allowed, so any existing options on teams which do not allow bulk transfers will now cease to work. Just want to check this is the desired behaviour (and have I understood the change correctly)?